src/Send/Send.js

```diff
@@ -4,30 +4,28 @@
 // SPDX-License-Identifier: MIT
 
 import React, { Component } from 'react';
-import { defaultAccount$, post$ } from '@parity/light.js';
-import { Redirect } from 'react-router-dom';
-import { toWei } from '@parity/api/lib/util/wei';
+import { balanceOf$, defaultAccount$ } from '@parity/light.js';
+import { map, switchMap } from 'rxjs/operators';
+import { fromWei, toWei } from '@parity/api/lib/util/wei';
 
 import ethereumIcon from '../assets/img/tokens/ethereum.png';
 import light from '../hoc';
 
 @light({
+  balance: () =>
+    defaultAccount$().pipe(
+      switchMap(balanceOf$),
+      map(value => +fromWei(value.toString()))
```

Comment:
@jacogr That's the way end-developers would need to access the current defaultAccount's balance. I.e. they need to `import { map, switchMap } from 'rxjs/operators'` themselves, which is indeed kind of a bummer. But I think it's the rx-way-of-things to import yourself what you need, so I guess I'll keep this isntead of a big fat object.

PS:
```javascript
@light({
  balance: ownProps => balanceof$(ownProps.address)
});
```
is possible too but requires the parent component to pass down address prop.